repos/spack_repo/builtin/packages/meep/package.py

```diff
@@ -83,7 +84,9 @@ class Meep(AutotoolsPackage):
     def configure_args(self):
         spec = self.spec
 
-        config_args = ["--enable-shared"]
+        config_args = ["LDFLAGS=-L{0}".format(spec["fftw-api"].prefix.lib)]
```

Comment:
@samfux84 That should be:
```suggestion
        config_args = ["LDFLAGS={0}".format(spec["fftw-api"].libs.ld_flags)]
```
Beware of typos, cause I didn't try to build.

Reply:
Thank you for the hint. I will test after the weekend and if meep builds fine, then I will commit your change.

Reply:
I tested it with the change and meep builds fine.

Reply:
Commited your proposed changes